release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4400,6 +4400,28 @@ $ oc adm release info 4.13.51 --pullspecs
 
 * Previously, Open vSwitch (OVS) could function by using older configurations after a cluster upgrade operations. These older configurations might have caused issues for networking components that interact with OVS. With this release, OVS is restarted after an upgrade operation so that OVS is forced to receive the latest configurations. (link:https://issues.redhat.com/browse/OCPBUGS-41723[*OCPBUGS-41723*])
 
+[id="ocp-4-13-52-bug-fixes"]
+==== Bug fixes
+
+* Previously, a group ID was not added to the `/etc/group` in the container when the `spec.securityContext.runAsGroup` attribute was set in the pod resource. With this release, this issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-41247[*OCPBUGS-41247*])
+
 [id="ocp-4-13-51-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.13.52
+[id="ocp-4-13-52"]
```

Comment:
I see the Extras advisory lists two bugs: https://errata.devel.redhat.com/advisory/139778. One bug does not need a release note but https://issues.redhat.com/browse/OCPBUGS-23018 does not give any indication. Would you be OK to contact the Assignee on this bug and confirm that status of whether a release note is needed? Most bugs exist on the Images advisory but sometimes other advisories might list a bug or two (rare, but it does happen).